src/api/userMeta/meta.crud.ts

```diff
@@ -0,0 +1,26 @@
+import { UserMeta } from '~/database/models/UserMeta';
+
+export interface UserMeta {
+  id: string;
+  createdAt: string;
+  updatedAt: string;
+  userId: string;
+  isWelcomeFlowPassed: boolean;
+}
+
+type CreateUserMetaPayload = Omit<UserMeta, 'id' | 'createdAt' | 'updatedAt'>;
+
+export type UserMetaResult = Omit<UserMeta, 'userId'>;
+
+export class UserMetaCrud {
+  static async saveMeta(payload: CreateUserMetaPayload) {
+    return UserMeta.create({
+      isWelcomeFlowPassed: payload.isWelcomeFlowPassed,
+      userId: payload.userId,
+    });
+  }
+
+  static async getMetaByUserId(userId: string) {
+    return UserMeta.findAll({ where: { userId } });
```

Comment:
I`ve changed the method findAll to find because we should have only one entry for one specific user